.make/test.mk

```diff
@@ -429,6 +425,22 @@ $(COV_PATH_INTEGRATION): $(SOURCES) $(GOCOVMERGE_BIN)
 	$(foreach package, $(TEST_PACKAGES), $(call test-package,$(TEST_NAME),$(package),$(COV_PATH_INTEGRATION),$(ERRORS_FILE),F8_RESOURCE_DATABASE=1 ADMIN_RESOURCE_UNIT_TEST=0,$(ALL_PKGS_COMMA_SEPARATED)))
 	$(call check-test-results,$(ERRORS_FILE))
 
+# -------------------------------------------------------------------
+# Generate mocks 
+# -------------------------------------------------------------------
+
+$(MINIMOCK_BIN):
+	@echo "building the minimock binary..."
+	@cd $(VENDOR_DIR)/github.com/gojuno/minimock/cmd/minimock && go build -v minimock.go
+
+.PHONY: generate-mocks
+## Generate mocks
+generate-mocks: deps $(MINIMOCK_BIN) 
+	@echo "Generating mocks..."
+	-mkdir -p test/generated/configuration
+	$(MINIMOCK_BIN) -i vendor/github.com/fabric8-services/fabric8-common/token.ManagerConfiguration -o ./test/generated/configuration/manager_configuration_mock.go -t ManagerConfigurationMock
```

Comment:
Add test/generated to clean target as well?

Reply:
ok, let me add that